lib/src/select/Select.stories.tsx

```diff
@@ -0,0 +1,543 @@
+      </svg>
+    ),
+  },
+  {
+    label: "Settings backup restore (just for previous configuration)",
+    value: "4",
+    icon: (
+      <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="currentColor">
+        <path d="M0 0h24v24H0V0z" fill="none" />
+        <path d="M14 12c0-1.1-.9-2-2-2s-2 .9-2 2 .9 2 2 2 2-.9 2-2zm-2-9c-4.97 0-9 4.03-9 9H0l4 4 4-4H5c0-3.87 3.13-7 7-7s7 3.13 7 7-3.13 7-7 7c-1.51 0-2.91-.49-4.06-1.3l-1.42 1.44C8.04 20.3 9.94 21 12 21c4.97 0 9-4.03 9-9s-4.03-9-9-9z" />
+      </svg>
+    ),
+  },
+];
+const url_options = [
+  {
+    label: "Social Media",
+    options: [
+      {
+        label: "Instagram",
+        value: "instagram",
+        icon: "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Instagram_logo_2016.svg/800px-Instagram_logo_2016.svg.png",
+      },
+      {
+        label: "Twitter",
+        value: "twitter",
+        icon: "https://cdn.computerhoy.com/sites/navi.axelspringer.es/public/styles/480/public/media/image/2013/08/17981-logo-twitter.png?itok=dElA6iAV",
+      },
+      {
+        label: "Facebook",
+        value: "facebook",
+        icon: "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b8/2021_Facebook_icon.svg/2048px-2021_Facebook_icon.svg.png",
+      },
+      {
+        label: "Pinterest",
+        value: "pinterest",
+        icon: "https://cdn-icons-png.flaticon.com/512/145/145808.png",
+      },
+    ],
+  },
+  {
+    label: "Design",
+    options: [
+      {
+        label: "Figma",
+        value: "figma",
+        icon: "https://logowik.com/content/uploads/images/figma.jpg",
+      },
+      {
+        label: "Adobe XD",
+        value: "adobexd",
+        icon: "https://cdn.worldvectorlogo.com/logos/adobe-xd-2.svg",
+      },
+      {
+        label: "Sketch",
+        value: "sketch",
+        icon: "https://cdn.worldvectorlogo.com/logos/sketch-2.svg",
+      },
+    ],
+  },
+];
+const optionsWithElipsisSmall = [
+  { label: "Optiond1234567890123456789", value: "1" },
+  { label: "Optiond12345678901234567890", value: "2" },
+  { label: "Option 03", value: "3" },
+  { label: "Option 041111112222222345", value: "4" },
+];
+const optionsWithElipsisMedium = [
+  { label: "Optiond1234567890123456789012345678901234", value: "1" },
+  { label: "Optiond12345678901234567890123456789012345", value: "2" },
+  { label: "Option 031111111111111111111111111111222", value: "3" },
+  { label: "Option 03111111111111111111111111111122", value: "4" },
+];
+const optionsWithElipsisLarge = [
+  { label: "Optiond1234567890123456789012345678901234567890123456789", value: "1" },
+  { label: "Optiond12345678901234567890123456789012345678901234567890", value: "2" },
+  { label: "Option 031111111111111111111111111111222345678901234567", value: "3" },
+  { label: "Option 04111111222222234", value: "4" },
+];
+export const Chromatic = () => (
+  <>
+    <ExampleContainer pseudoState="pseudo-hover">
```

Comment:
It may be interesting to change this example to multiple with a selected option in order to also test `:hover` state of the SelectionIndicator clear, what do you think?

![image](https://user-images.githubusercontent.com/44321109/152778832-ee67bd0d-0c60-4506-af71-ededd436a070.png)

Reply:
Same as above